packages/target-ethers-v4-test/types/Events.d.ts

```diff
@@ -41,6 +43,8 @@ interface EventsInterface extends Interface {
       encodeTopics([value1, value2]: [boolean | null, null]): string[];
     }>;
 
+    Event4: TypedEventDescription<{ encodeTopics([data]: [null]): string[] }>;
```

Comment:
`data` of type `null`? That doesn't sound right 😄

Reply:
or is this case just unsupported in ethers v4?

Reply:
Yeah I think it might not work with `ethers-v4`. Dunno - we don't care too much about that target.